cmd/chaTer-server/main.go

```diff
@@ -0,0 +1,12 @@
+package main
```

Comment:
1. Let's not call it cmd directory. In our go projects we usually have a different meaning for it. Make it cli or something more explicit. Also what's with the main package here? afaiu there should be only 1 main and we are messing that up

Reply:
Check the link I posted in the PR description. I followed that.

After reviewing that article, let me know what to change it to.

Reply:
Check previous comment